hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileBlock.java

```diff
@@ -1601,7 +1601,6 @@ public HFileBlock readBlockData(long offset, long onDiskSizeWithHeaderL, boolean
                        offset + " filesize " + fileSize +
                        " but this cannot happen because doVerify is " +
                        doVerificationThruHBaseChecksum;
-          HFile.LOG.warn(msg);
```

Comment:
oof that is bad logging practice.

does the IOException get properly handled and logged?